hack/lib/build/environment.sh

```diff
@@ -173,12 +175,21 @@ function os::build::environment::run() {
   if [[ -z "${volume}" ]]; then
     volume="origin-build-$( git rev-parse "${commit}" )"
   fi
+  volume="$( echo "${volume}" | tr '[:upper:]' '[:lower:]' )"
+  export OS_BUILD_ENV_VOLUME="${volume}"
+
+  if [[ -n "${OS_BUILD_NEW_VOLUME:-}" ]]; then
+    if docker volume inspect "${volume}" >/dev/null 2>&1; then
```

Comment:
`>/dev/null` is by itself a valid expression. It does make use of the redirect `>` assuming fd `1` if nothing is given, but it's mostly clear. It's like three leaps to prepend this valid expression with `&`, a char that is used to indicate file descriptors in a redirection target ... only here you put it on the redirection source ... and it doesn't default to `1` as usual but to `2`? The argument is made in a million different ways in Bash -- and honestly also in languages like Ruby. If there are ten million different ways to do something, all relatively equal in the eyes of the machine, I want to choose the one that is the simplest and requires the least syntactic sugar and has the lower barrier to entry. `&>/dev/null` is not that.